utils/utils.go

```diff
@@ -0,0 +1,82 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/syndtr/gocapability/capability"
+)
+
+// CapValid checks whether a capability is valid
+func CapValid(c string, hostSpecific bool) error {
```

Comment:
This (and `UnitListValid`) are both validators.  Why not keep them in the validate package?